api/src/http.rs

```diff
@@ -404,6 +404,7 @@ pub struct ProxyConfig {
     pub fallback: bool,
     /// Interval of P2P proxy health checking, in seconds.
     pub check_interval: u64,
+    pub use_http: bool,
 }
```

Comment:
Please add doc for the new "use_http" field

Reply:
Added